interface/schema.graphql

```diff
@@ -7,7 +7,7 @@ type Module {
   tryResolveUri(
     authority: String!
     path: String!
-  ): MaybeUriOrManifest @env(required: false)
```

Comment:
These changes should be reverted no? I'm good with this PR if we aren't changing the interface.

Reply:
done